target/linux/ar71xx/image/generic.mk

```diff
@@ -722,6 +722,15 @@ define Device/xd3200
 endef
 TARGET_DEVICES += xd3200
 
+define Device/xd6700
+  DEVICE_TITLE := YunCore XD6700
+  DEVICE_PACKAGES := kmod-ath10k ath10k-firmware-qca9887
+  BOARDNAME = XD6700
+  CONSOLE = ttyS0,115200
```

Comment:
`ttyS0,115200` is default value for `CONSOLE` in whole ar71xx target, you can drop this line.